app/src/main/java/com/android/partagix/ui/screens/InventoryCreateOrEditItemScreen.kt

```diff
@@ -200,13 +209,19 @@ fun InventoryCreateOrEditItem(
                 readOnly = false)
 
             Spacer(modifier = modifier.height(8.dp))
-
-            OutlinedTextField(
-                value = uiLocation.toString(), // TODO: get default or user's location
-                onValueChange = { it -> uiLocation = Location(it) },
-                label = { Text("Where") },
-                modifier = modifier.fillMaxWidth(),
-                readOnly = false)
+            /*
```

Comment:
this comment seems useless and there is a TO_DO left that seems to be done so we want to erase it